presentation/src/main/java/team/jsv/icec/ui/main/MainActivity.kt

```diff
@@ -20,10 +26,53 @@ class MainActivity :
     BaseActivity<ActivityMainBinding>(R.layout.activity_main) {
 
     private val viewModel: MosaicViewModel by viewModels()
+    private lateinit var onBackPressedCallback: OnBackPressedCallback
+    private val navController by lazy {
+        (supportFragmentManager.findFragmentById(R.id.fragmentContainerView) as NavHostFragment).findNavController()
+    }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        requestPermissions(PermissionUtil.getPermissions())
+        initTopBar()
+        initView()
+        setOnBackPressedCallback()
+    }
+
+    override fun onResume() {
+        super.onResume()
+
+        initClickListeners()
+    }
+
+    private fun initTopBar() {
+        viewModel.screenStep.observe(this) {
+            when (it) {
+                ScreenStep.SelectMosaicEdit -> {
+                    binding.topBar.buttonBack.visibility = View.VISIBLE
```

Comment:
fun View.visible(){
    visibility = View.VISIBLE
}

fun View.gone(){
    visibility = View.GONE
}
위 확장 함수를 하나 만들어서 사용하면, 선언형으로 더 간결하게 표현할 수 있을 것 같네요!

```suggestion
                    binding.topBar.visible()
```

Reply:
[96fea1a](https://github.com/JobSeek-Void/ic-ec/pull/18/commits/96fea1adcedd4d3b8a2632cdd153fe26a0f513d8) 리뷰 반영 완료